cli/crates/cli/src/cli_input.rs

```diff
@@ -213,6 +216,50 @@ pub struct LogsCommand {
     pub no_follow: bool,
 }
 
+#[derive(Debug, clap::Args)]
+pub struct StartCommand {
+    /// Use a specific port
+    #[arg(short, long, default_value_t = DEFAULT_PORT)]
+    pub port: u16,
+    /// Log level to print from function invocations, defaults to 'log-level'
+    #[arg(long, value_name = "FUNCTION_LOG_LEVEL")]
+    pub log_level_functions: Option<LogLevelFilter>,
+    /// Log level to print for GraphQL operations, defaults to 'log-level'
+    #[arg(long, value_name = "GRAPHQL_OPERATION_LOG_LEVEL")]
+    pub log_level_graphql_operations: Option<LogLevelFilter>,
+    /// Log level to print for fetch requests, defaults to 'log-level'
+    #[arg(long, value_name = "FETCH_REQUEST_LOG_LEVEL")]
+    pub log_level_fetch_requests: Option<LogLevelFilter>,
+    /// Default log level to print
+    #[arg(long)]
+    pub log_level: Option<LogLevelFilter>,
+    #[arg(long)]
+    pub listen: Option<IpAddr>,
```

Comment:
I'd possibly name this host or something more indicative

Reply:
well it's not really a host, in PostgreSQL they're naming it `listen_addresses`. We could name it `address` or `ip` eventually. wdyt?

Reply:
`address` is probably better IMO but you could poll it

Reply:
Do other CLIs use `listen`?

Reply:
let's go with address then :)

Reply:
I'd poll it